src/libraries/Directory.Build.targets

```diff
@@ -16,6 +16,11 @@
   <Import Project="$(RepositoryEngineeringDir)resources.targets" />
   <Import Project="..\..\Directory.Build.targets" />
 
+  <!-- To suppress warnings about reseting the assembly version.-->
+  <PropertyGroup Condition="'$(AutoGenerateAssemblyVersion)' == 'true'">
+    <AssemblyVersion></AssemblyVersion>
```

Comment:
I see that this is the warning you're avoiding:
https://github.com/dotnet/arcade/blob/20daa7891e53bd30fc12640fad4a0a9de15ec953/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.targets#L15

I see that you have to do this, because our default AssemblyVersion is set in props, before a project might set `AutoGenerateAssemblyVersion`.

So this allows a project to override the repo wide AssemblyVersion with an automatically set AssemblyVersion.  I guess an alternative would be to have the repo-wide AssemblyVersion be set in targets and consider if some project set AutoGenerateAssemblyVersion.

Reply:
i didnt do that because this needs to be ported to 6.0 and don't want to cause any side-effects by moving the assembly version to targets file. There shouldnt be any but didnt want to take unnecessary risk.